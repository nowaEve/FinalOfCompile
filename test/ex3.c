void main() { 
    sleep(3000);
}