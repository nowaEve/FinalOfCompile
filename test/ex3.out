24 19 0 5 25 0 3000 31 21 0